definition_validator/validator.cc

```diff
@@ -181,6 +181,14 @@ void validateOverriding(const core::GlobalState &gs, core::SymbolRef method) {
             }
         }
         auto isRBI = absl::c_any_of(method.data(gs)->locs(), [&](auto &loc) { return loc.file().data(gs).isRBI(); });
+        if (!method.data(gs)->isOverride() && method.data(gs)->hasSig() && overridenMethod.data(gs)->isOverridable() &&
+            overridenMethod.data(gs)->hasSig() && !isRBI) {
```

Comment:
I don't think we need `isRBI` here as it's unlikely that we have `overridable` rbis

Reply:
I got rid of `isRBI` for the `overridable` case, but kept it for the `implementation` case, because of `Enumerable#each` (which is abstract but shows up in a _lot_ of RBIs.)